### ISSUE TITLE
Copying code to existing ESPHome setup

### ISSUE THREAD

**Issue by Zero-Circle** [patrickcollins12/esphome-fan-controller#41]
Hi guys, silly question Im sure but im fairly new to ESPHome...

Am I able to copy the yaml manually to an existing esphome setup?
I will remove the top 40 rows that dont relate to my setup.
Is this an adequate way of doing this or am i missing something fundamental?

Thankyou all for the support of this cool project!

**Comment by patrickcollins12** [comment]
Yes you can run multiple configs on a single esphome install. Don't delete the stuff at the top though.

**Comment by patrickcollins12** [comment]
If you mean running this fan controller on an existing ESP32, yes you can do that. They're only $4 though, why would you dual purpose them?

**Comment by Zero-Circle** [comment]
Hi mate, i figured it out. I needed to also control LEDs and RGB control light for my 3d printer enclosure.

**Comment by patrickcollins12** [comment]
Cool. Can you share some photos and a screenshot?
